src/pages/gen2/build-a-backend/auth/override-cognito/index.mdx

```diff
@@ -21,12 +21,65 @@ import { defineAuth } from '@aws-amplify/backend';
 export const auth = defineAuth({
   loginWith: {
     email: true
-  }
+  },
 });
 ```
 
 You can customize your authentication resource to enable it to behave exactly as needed for your use cases by modifying it directly using [AWS Cloud Development Kit (CDK)](https://aws.amazon.com/cdk/)
 
+## Customize Cognito UserPool email verification settings
+
+You can change the verification email by overriding the `verificationEmailStyle` and the `verificationEmailBody`.
+
+```ts title="amplify/auth/resource.ts"
+import { defineAuth } from "@aws-amplify/backend";
+
+export const auth = defineAuth({
+  loginWith: {
+    email: {
+      verificationEmailStyle: "CODE",
+      verificationEmailBody: (code: string) =>
+        `Welcome! Your verification code is ${code}.`,
+      verificationEmailSubject: "Welcome! Here is your verification code",
+    },
+  },
+});
+```
```

Comment:
I like this example as a form of "customizing sign-up flow", can we lift this to https://github.com/aws-amplify/docs/blob/next-release/gen2/src/pages/gen2/build-a-backend/auth/enable-sign-up/index.mdx ?

Reply:
OK I moved it to the `enable-sign-up` section